src/frontend/js/components/Badge/index.tsx

```diff
@@ -6,6 +6,7 @@ type BadgeProps = PropsWithChildren<{
 }>;
```

Comment:
In the commit body
```diff
- Contract navigation link have a specific behavior about they query
+ Contract navigation link has a specific behavior about query
params and a possible count badge.
This logic was part of the OrganizationSidebar component but need's to
be implemented in the Course section.

- To avoid duplicating code, we extrat contract's dedicated code in a new
+ To avoid duplicating code, we extract contract's dedicated code in a new
ContractNavLink component.
```